### ISSUE TITLE
Opening modeless (= non modal) dialogs/windows

### ISSUE THREAD

**Issue by giuliohome** [ReedCopsey/Gjallarhorn.Bindable#18]
Hello!
First of all, congrats, this is a nice project, quite complex for me (I'm used to MVVM but not to Elmish approach).
I'm mentioning a [sample](https://github.com/ReedCopsey/Gjallarhorn.Bindable/tree/master/samples/Collections/WpfCollectionManager) of `Gjallarhorn.Bindable` after a long trail in a closed [Elmish.WPF issue](https://github.com/elmish/Elmish.WPF/issues/24#issuecomment-419713904).
The only reason why I am disturbing you is that users would obviously prefer to have the possibility to open two items at the same time so that they could possibly compare them. 
Anyway from a technical standpoint I am looking at the `Nav.fs` under the project `Gjallarhorn.Bindable.Wpf` and specifically at [the code lines](https://github.com/ReedCopsey/Gjallarhorn.Bindable/blob/86bcf6dd65d0be970ee4b18a02f2e3c7b7a1c7df/src/Gjallarhorn.Bindable.Wpf/Nav.fs#L139-L141)

```
        | ModalDialog window ->
            window.Owner <- mainWindow
            window.ShowDialog() |> ignore
```

I've done a very quick, dirty and rough change by replacing `ShowDialog` with `Show` and it looks like it would work as expected so maybe you might want to add an enhancement to open new modeless windows? I'd be very happy, but thanks anyway!

**Comment by ReedCopsey** [comment]
Yes, that works fine. The Collections sample shows multiple modeless windows already, BTW.

**Comment by giuliohome** [comment]
@ReedCopsey 
Sorry, I'm lost: where does it already show multiple non-modal windows? I tried the edit button and it opens a modal dialog... what am I missing? Thank you!

**Comment by ReedCopsey** [comment]
Login to main window switches modeless windows...

**Comment by giuliohome** [comment]
I fail to see what you mean.
My steps here

1. I run `WpfCollectionManager`
2. I have to pass a login dialog where with username 3 or more characters and password "Gjallarhorn" and aside from that, I don't see any "switch" here but only a "Submit" button
3. only after the login is passed with 2 checkboxes "Add new requests" and "Process handled requests" but no one has the effect of changing the edit/info window from modal to modeless AFAICS

Maybe I'm completely missing the meaning of your answer and I'm sorry for that.
